libs/fixtures/data/electionFamousNames2021/cdf-election.ts

```diff
@@ -0,0 +1,244 @@
+      ElectionScopeId: 'state-hamilton', // The GPUnit for this election
+      StartDate: '2023-02-01',
+      EndDate: '2023-02-01',
+      Type: 'general', // Other types: primary, runoff, special, etc.
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'Lincoln Municipal General Election',
+          },
+        ], // Election name in other languages would go here
+      },
+
+      // Definition of all candidates in the election across contests
+      Candidate: [
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'sherlock-holmes',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Sherlock Holmes',
+              },
+            ],
+          },
+          PartyId: 'party-0',
+        },
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'thomas-edison',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Thomas Edison',
+              },
+            ],
+          },
+          PartyId: 'party-1',
+        },
+      ],
+
+      // Definition of contests
+      Contest: [
+        {
+          '@type': 'BallotDefinition.CandidateContest',
+          '@id': 'mayor',
+          ElectionDistrict: 'district-1',
+          // TODO what is the analogous field for "section"?
+          Name: 'Mayor', // Contest name, not for ballot
+          BallotTitle: {
+            // Display name for contest on ballot
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Mayor',
+              },
+            ],
+          },
+          VotesAllowed: 1,
+          // TODO there's no field analogous to "allowWriteIns" in CDF
+          // Contest options reference the candidates defined above by id
+          ContestOption: [
+            {
+              '@type': 'BallotDefinition.CandidateOption',
+              CandidateIds: ['sherlock-holmes'],
+            },
+            {
+              '@type': 'BallotDefinition.CandidateOption',
+              CandidateIds: ['thomas-edison'],
+            },
+          ],
+        },
+      ],
+
+      // Definition of ballot styles
+      BallotStyle: [
+        {
+          '@type': 'BallotDefinition.BallotStyle',
+          '@id': 'ballot-style-1',
+          // TODO do we also reference districts here? what's the relationship between districts and precincts?
+          GpUnit: ['precinct-23', 'precinct-22'],
```

Comment:
In the VX election def, ballot styles are associated with precincts and districts. Why is that? Do we need to preserve that in CDF?

Reply:
In CDF-land, precincts and districts are all `GpUnit`s, and I've also been wondering how we handle. This seems like one of the thornier issues. I wonder if we want to move to a similar model internally, but that would be a huge change. After all, say you have a mail-in state. Not all ballots are associated with a precinct. But in our system, all ballots are always associated with a precinct. 

We use `districts` to determine which contests are on a ballot style, because we don't have any idea of composing `GpUnits`. For us, `precincts` is basically a reporting split and a label in the bottom the ballot - it doesn't give any information about what contests to put on a ballot. So, we would need to maintain the data somehow or change how we handle this internally.

Reply:
I agree that the geographies are probably the trickiest bit here that we should probably discuss, but as mentioned in another comment, I think we're going to want to move to a model where we aren't constrained to the fix district/precinct hierarchy only. For reporting, we'll need to be able to cut by all geographic associations.